packages/builder/src/components/automation/AutomationBuilder/FlowChart/ExternalActions.js

```diff
@@ -0,0 +1,13 @@
+import DiscordLogo from "assets/discord.svg"
+import ZapierLogo from "assets/zapier.png"
+import IntegromatLogo from "assets/integromat.png"
+import SlackLogo from "assets/slack.svg"
+import n8nlogo from "assets/n8n.png"
+
+export const externalActions = {
```

Comment:
Two things on this:
1. Right now we don't have an n8n action
2. I added a `internal: false | true` to the action definitions (was one of the last things that I did) - might be useful for determining the external actions!

Reply:
Not quite sure why I added that haha, maybe because it was in the design, it won't show up in the UI as there was no backend for it, but i'll remove it anyway. And ah that;s very nice I hadn't noticed that flag, that'll actually simplify that area a good bit.